templates/link_template.function_app.setup.json

```diff
@@ -113,7 +120,8 @@
         }
     },
     "variables": {
-        "enableFortiAnalyzer": "[equals(parameters('FortiAnalyzerIntegrationOptions'), 'yes')]"
+        "enableFortiAnalyzer": "[equals(parameters('FortiAnalyzerIntegrationOptions'), 'yes')]",
+        "terminateUnhealthyVM": "[equals(parameters('TerminateUnhealthyVM'), 'yes')]"
```

Comment:
if it is changed here in: https://github.com/fortinet/fortigate-autoscale-azure/pull/74/files#diff-f85082b086299768e0319777800bea85138d26db31e15a94fae798580c030c98R237 we do not need this line
```suggestion
```